.github/workflows/build-standalone-on-release.yaml

```diff
@@ -5,9 +5,6 @@ on:
       - v[1-9].*
 
 jobs:
-  build-validation:
-    name: Build Test
-    uses: ./.github/workflows/run-lint.yaml
```

Comment:
We can remove the validation here.
https://github.com/harvester/harvester-ui-extension/blob/1533ea070a7d89e8bae9f23432b2d1cd153c553d/.github/workflows/build-standalone-on-release.yaml#L11-L12